src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -143,12 +147,23 @@ public function __construct(array $options = [])
 
         $options['error_handler'] ??= SymfonyErrorHandler::class;
 
+        $workerLoopMax = $options['worker_loop_max'] ?? $_SERVER['FRANKENPHP_LOOP_MAX'] ?? $_ENV['FRANKENPHP_LOOP_MAX'] ?? null;
+        if (null !== $workerLoopMax && (!\is_numeric($workerLoopMax) || $workerLoopMax <= 0)) {
+            throw new \LogicException(\sprintf('The "worker_loop_max" runtime option must be a positive integer, "%s" given.', $workerLoopMax));
+        }
+
+        $options['worker_loop_max'] = (int) $workerLoopMax ?? 500;
```

Comment:
cast has higher precedence than ?? so this is likely wrong

Reply:
Ah yes good catch, thanks